admin/config.js

```diff
@@ -53,12 +53,13 @@ module.exports = {
     }
   },
   Logging: {
+    // @TODO: Remove hardcoded key
     LogDna: {
-      key: process.env.LOGDNA_API_KEY,
+      key: '5a3d75a421e412afe3139e4a02a6ce19', // process.env.LOGDNA_API_KEY,
       hostname: `${os.hostname()}:${process.pid}`,
       ip: undefined,
       mac: undefined,
-      app: `MTCAdmin:${getEnvironment()}`,
+      app: `MTC-${getEnvironment()}`,
```

Comment:
why remove the admin part?  this will be useful when we have other applications (such as the APIs) logging to the same source

Reply:
It will be restored in the next push.